content/patterns/rag-llm-gitops/getting-started.md

```diff
@@ -109,10 +114,10 @@ global:
     useCSV: false
     syncPolicy: Automatic
     installPlanApproval: Automatic
-# Possible value for db.type = [REDIS, PGVECTOR]
+# Possible value for db.type = [REDIS, EDB]
   db:
     index: docs
-    type: PGVECTOR  <--- Default is PGVECTOR, Change the db type to REDIS for REDIS deployment
+    type: EDB  <--- Default is EDB, Change the db type to REDIS for Redis deployment
```

Comment:
If this is intended to be functional, you need to comment out the `<--- ...` part or yaml will error.

Reply:
I just used the one in the pattern, understanding it wasn't meant to be functional. But I committed your change. Thanks.